themes/default/views/admin/orders/show.blade.php

```diff
@@ -2,249 +2,282 @@
+                                    <div id="tooltip-animation6" role="tooltip" class="absolute z-10 invisible inline-block px-3 py-2 text-sm font-medium text-white transition-opacity duration-300 bg-gray-900 rounded-lg shadow-sm opacity-0 tooltip dark:bg-gray-700">
+                                        {{__('Mark as paid')}}
+                                        <div class="tooltip-arrow" data-popper-arrow></div>
+                                    </div>
+                                </div>
+                            @endif
+
+                            @if ($order->products[0]->status == 'suspended')
+                                <div>
+                                    <form method="POST" action="{{ route('admin.orders.unsuspend', $order->id) }}">
+                                        @csrf
+                                        <button class="bg-[#4EB165] button button-success" type="submit" data-tooltip-target="tooltip-animation3">
+                                            <i class="ri-lock-unlock-line"></i>
+                                        </button>
+                                    </form>
+                                    <div id="tooltip-animation3" role="tooltip" class="absolute z-10 invisible inline-block px-3 py-2 text-sm font-medium text-white transition-opacity duration-300 bg-gray-900 rounded-lg shadow-sm opacity-0 tooltip dark:bg-gray-700">
+                                        {{__('Unsuspend product')}}
+                                        <div class="tooltip-arrow" data-popper-arrow></div>
+                                    </div>
+                                </div>
+                            @elseif($order->products[0]->status == 'paid')
+                                <div>
+                                    <form method="POST" action="{{ route('admin.orders.suspend', $order->id) }}">
+                                        @csrf
+                                        <button class="bg-[#DC3A3A] button button-danger" type="submit" data-tooltip-target="tooltip-animation4">
+                                            <i class="ri-lock-2-line"></i>
+                                        </button>
+                                    </form>
+                                    <div id="tooltip-animation4" role="tooltip" class="absolute z-10 invisible inline-block px-3 py-2 text-sm font-medium text-white transition-opacity duration-300 bg-gray-900 rounded-lg shadow-sm opacity-0 tooltip dark:bg-gray-700">
+                                        {{__('Suspend product')}}
+                                        <div class="tooltip-arrow" data-popper-arrow></div>
+                                    </div>
+                                </div>
+                            @endif
+
+                            <div>
+                                <form> <!-- TODO: DELETE ONLY THIS SERVICE, NOT ORDER route('admin.orders.delete', $order->id) -->
```

Comment:
You should not add non functional UI elements

Reply:
This version is not branch for production use.
This will be added properly before the next version is released :)